azext_iot/_params.py

```diff
@@ -121,6 +123,11 @@ def load_arguments(self, _):
         context.argument('content_type', options_list=['--content-type', '--ct'],
                          help='Specify the Content-Type of the message payload to automatically format the output to that type.')
         context.argument('device_query', options_list=['--device-query', '-q'], help='Specify a custom query to filter devices.')
+        context.argument('interface', options_list=['--interface', '-i'],
```

Comment:
Please see my below comments regarding the elements.

Reply:
fixed as mentioned in comments